### PR TITLE
example bunch: use incident filed laser

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- remove laser.param
- configure incidnet field laser equal to the old laser in laser.param
  - `png.param` normalize field values based on incident filed laser amplitude

- [x] rebase against #4457
- [x] run a tests

# bunch example (step 700)
`./bin/picongpu -d 1 1 1 -g 128 1024 128 -s 2000 --periodic 1 0 1 --e_png.period 100 --e_png.axis yx --e_png.slicePoint 0.5 --e_png.folder pngElectronsYX`

e_png_yx_0 5_000700

You can see that boundaries are not periodic, the field must be equal in the transversal direction.

### CODE REVIEW

**Comment by psychocoderHPC** [comment]
After rebasing against #4457 all works like expected

# step 700:
e_png_yx_0 5_000700

**Comment by psychocoderHPC** [comment]
@steindev PR can be reviewed

File: share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+#pragma once
+
+#include "picongpu/fields/incidentField/profiles/profiles.def"
+
+#ifndef PARAM_A0
+#    define PARAM_A0 8.0
+#endif
+
+#ifndef PARAM_WAVE_LENGTH_SI
+#    define PARAM_WAVE_LENGTH_SI 0.8e-6
```
Comment (PrometheusPi) [inline share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param]: is not used below - please remove

File: share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+ * constexpr int32_t POSITION[3][2] = { {16, -16}, {16, -16}, {16, -16} };
+ * @endcode
+ */
+
+#pragma once
+
+#include "picongpu/fields/incidentField/profiles/profiles.def"
+
+#ifndef PARAM_A0
+#    define PARAM_A0 8.0
```
Comment (PrometheusPi) [inline share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param]: is not used below, please remove

File: share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+#endif
+
+#ifndef PARAM_WAVE_LENGTH_SI
+#    define PARAM_WAVE_LENGTH_SI 0.8e-6
+#endif
+
+#ifndef PARAM_PULSE_LENGTH_SI
+#    define PARAM_PULSE_LENGTH_SI 5.e-15
```
Comment (PrometheusPi) [inline share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param]: is not used below, please remove

File: share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+                static constexpr float_64 AMPLITUDE_SI = _A0 * UNITCONV_A0_to_Amplitude_SI;
+
+                /** Pulse length: sigma of std. gauss for intensity (E^2)
+                 *  PULSE_LENGTH_SI = FWHM_of_Intensity   / [ 2*sqrt{ 2* ln(2) } ]
+                 *                                          [    2.354820045     ]
+                 *  Info:             FWHM_of_Intensity = FWHM_Illumination
+                 *                      = what a experimentalist calls "pulse duration"
+                 *  unit: seconds (1 sigma)
+                 */
+                static constexpr float_64 PULSE_LENGTH_SI = 10.615e-15 / 4.0;
```
Comment (PrometheusPi) [inline share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param]:
```suggestion
                static constexpr float_64 PULSE_LENGTH_SI = 2.65e-15;

```
old code reads (it is close but not exact)

File: share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+                 *
+                 * unit: m
+                 *
+                 * Note the plane wafe does not have a focus position therefore it is set to zero for all directions.
+                 *
+                 * @{
+                 */
+                static constexpr float_64 FOCUS_POSITION_X_SI = 0.0;
+                static constexpr float_64 FOCUS_POSITION_Y_SI = 0.0;
+                static constexpr float_64 FOCUS_POSITION_Z_SI = 0.0;
```
Comment (PrometheusPi) [inline share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param:157]: Just a general question: why is there a focus position for the plane wave anyway? It has no meaning.

**Review by PrometheusPi — CHANGES_REQUESTED** [review]
Only minor housekeeping to remove unused pragmas

File: share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param (hunk 5 above)
Comment (psychocoderHPC) [reply share/picongpu/examples/Bunch/include/picongpu/param/incidentField.param:157]: because this is a requirement by the incident field laser

**Comment by psychocoderHPC** [comment]
updated

**Review by PrometheusPi — APPROVED** [review]
Thanks for the quick adjustment @psychocoderHPC

**Pull request merged**